zram-no-defaults.yaml

```diff
@@ -0,0 +1,5 @@
+remove-from-packages:
+  # zram-generator-0.3.2 (shipped in centOS 9) provides a default
+  # zram-generator config, we want to disable it
+  - - zram-generator
+    - "/usr/lib/systemd/zram-generator.conf"
```

Comment:
Nit for a future PR: Missing empty line at the end of the file

Reply:
I should add https://github.com/coreos/fedora-coreos-config/blob/testing-devel/ci/find-whitespace to this repo.